apps/dashboard/app/apps/ood_app.rb

```diff
@@ -181,13 +181,19 @@ def manifest_path
     path.join("manifest.yml")
   end
 
-  def icon_path
+  def icon_path_png
     path.join("icon.png")
   end
```

Comment:
We probably want to bury the knowledge about svgs here instead of adding 2+ interfaces. What I mean is, we want to keep the `icon_path` interface and return svg if it exists, png if it exists and nil if neither.

Reply:
actually, not nil probably `Pathname.new('')` because we're calling `file?` on the return value all over the place.